tests_homework/tests.py

```diff
@@ -0,0 +1,225 @@
+import unittest
+import random
+import uuid
+import logging
+from testfixtures import LogCapture
+from homework import Car, Cesar, Garage, Serialization
+from constants import TOWNS, CARS_TYPES, CARS_PRODUCER, CESAR_NAME
+
+
+def data_init_car():
+    return Car(
+        car_type=random.choice(CARS_TYPES),
+        producer=random.choice(CARS_PRODUCER),
+        price=round(random.uniform(1, 100000), 2),
+        mileage=round(random.uniform(1, 100000), 2)
+    )
+
+
+def data_init_garage():
+    cesar_id = uuid.uuid4()
+    return Garage(
+        cars=[data_init_car(), data_init_car(), data_init_car()],
```

Comment:
```suggestion
        cars=3*[data_init_car()],
```
це допоможе уникнути дублювання коду